docs/about-filecoin/wrapped-filecoin.md

```diff
@@ -0,0 +1,118 @@
+- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. They may have services always have a counter-party risk from their operator but could have some other advantages like insurance, guaranteed backing, support, etc.
+
+## Wrapping your FIL process
+
+There are several web-based applications for token wrapping. Here is an overview of the steps:
+
+1. **Choose your wrapped FIL type.** Depending on the chain where you want to use your wrapped FIL, different types are available.
+2. **Open the web app and connect your wallet.** Most apps require a web or mobile wallet.
+3. **Choose the amount.** Double-check the fee structure, which may vary by network and application.
+4. **Click to mint.** You may need to wait several minutes for the target network to confirm the transaction.
+
+For example, here’s how it works in RenBridge, a decentralized application (dApp) that allows users to mint real FIL (as well as BTC, ZEC, and more) on Ethereum as an ERC20 token (renFIL):
+
+- Asset selection process
+
+![renBridge asset selection window](./images/wrapped-filecoin/select-asset.png)
+
+- Fees and confirmation
+
+![renBridge fees and confitmation window](./images/wrapped-filecoin/fees-and-confirm.png)
+
+## Wrapping solutions comparison table
+
+<table class="comparison">
+    <tr>
+        <th>Name</th>
+        <th>Project</th>
+        <th>Infrastructure</th>
+        <th>Custodian</th>
+        <th>Main Blockchain</th>
+        <th>Link</th>
+    </tr>
+    <tr>
+        <td>renFIL</td>
+        <td>renBridge</td>
+        <td>Decentralized</td>
+        <td>Decentralized</td>
+        <td>Etherium, Binance, Polygon</td>
+        <td>https://bridge.renproject.io/mint</td>
+    </tr>
+    <tr>
+        <td>wFIL</td>
+        <td>WRAPPED</td>
+        <td>Centralized</td>
+        <td>Anchorage</td>
+        <td>Etherium</td>
+        <td>https://www.wrapped.com/</td>
+    </tr>
+    <tr>
+        <td>BFIL</td>
+        <td>Binance</td>
+        <td>Centralized</td>
+        <td>Binance</td>
+        <td>Etherium via Binance Token Canal </td>
+        <td>https://www.binance.com/</td>
+    </tr>
+    <tr>
+        <td>vFIL</td>
+        <td>Venus</td>
+        <td>Decentralized</td>
+        <td>Decentralized</td>
+        <td>Binance Smart Chain </td>
+        <td>https://www.venus.io/</td>
+    </tr>
+    <tr>
+        <td>eFIL</td>
+        <td>Gemini</td>
+        <td>Decentralized</td>
+        <td>Gemini</td>
+        <td>Etherium </td>
+        <td>https://www.gemini.com/</td>
+    </tr>
+    <tr>
+        <td>HFIL</td>
+        <td>Huobi</td>
+        <td>Decentralized</td>
+        <td>Huobi</td>
+        <td>Etherium</td>
+        <td>https://www.huobi.com</td>
+    </tr>
+</table>
```

Comment:
Tables should follow the markdown syntax: https://www.markdownguide.org/extended-syntax/#tables

Reply:
I actually used the same table format as in https://github.com/filecoin-project/filecoin-docs/blob/master/docs/about-filecoin/filecoin-compared-to.md, just updated it to my needs. Will shortly update it to the markdown syntax

Reply:
```suggestion
| **Name** | **Project** | **Infrastructure** | **Custodian** | **Main Blockchain**              | **Link**                            |
|----------|-------------|--------------------|---------------|----------------------------------|-------------------------------------|
| renFIL   | renBridge   | Decentralized      | Decentralized | Etherium, Binance, Polygon       | https://bridge\.renproject\.io/mint |
| wFIL     | WRAPPED     | Centralized        | Anchorage     | Etherium                         | https://www\.wrapped\.com/          |
| BFIL     | Binance     | Centralized        | Binance       | Etherium via Binance Token Canal | https://www\.binance\.com/          |
| vFIL     | Venus       | Decentralized      | Decentralized | Binance Smart Chain              | https://www\.venus\.io/             |
| eFIL     | Gemini      | Decentralized      | Gemini        | Etherium                         | https://www\.gemini\.com/           |
| HFIL     | Huobi       | Decentralized      | Huobi         | Etherium                         | https://www\.huobi\.com             |
```

This works better?